### PR TITLE
Add v15 network upgrade blog post

### PR DESCRIPTION
**Pull request by sethforprivacy**
~~Still awaiting image permission, but wanted to go ahead and open PR to ensure all else is good to go while I await that.~~

~~Please do not merge this until I have image permissions!~~

Edit: Permission granted, should be ready for review and merge now!

### CODE REVIEW

**Comment by SamsungGalaxyPlayer** [comment]
Should we commit to the name `Flourine Fermi` in the post as well? Better for branding if that's what we will go with.

> The new software will be released one month before the network upgrade, on the **~16th of June.**

to

> The new software (v0.18 "Flourine Fermi") will be released one month before the network upgrade, on the **~16th of June.**

https://github.com/monero-project/meta/issues/623

File: _posts/2022-04-19-network-upgrade-july-2022.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+The next network upgrade will take place on **16th July, 2022** (block 2,668,888). Network upgrades are technically hard-forks and require all participants in the Monero network to update their software. The new software will be released one month before the network upgrade, on the **~16th of June**.
+
+This network upgrade will introduce important new features like:
+
+- A ring-size increase from 11 to 16, increasing the base anonymity set (read: plausible deniability, or base privacy) of every transaction on the network
+- [Upgraded Bulletproofs algorithm, "Bulletproofs+"]({{ site.baseurl_root }}/2020/12/24/Bulletproofs+-in-Monero.html), which will **reduce the typical transaction size by ~5-7%** and **improve typical verification performance by ~5-7%**, making every transaction lighter and faster
+- [View tags, a brilliant way to reduce wallet sync times by 30-40%](https://localmonero.co/knowledge/view-tags-reduce-monero-sync-time)
```
Comment (SamsungGalaxyPlayer) [inline _posts/2022-04-19-network-upgrade-july-2022.md:20]: Bold

[View tags, a brilliant way to **reduce wallet sync times by 30-40%**](https://localmonero.co/knowledge/view-tags-reduce-monero-sync-time)

File: _posts/2022-04-19-network-upgrade-july-2022.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+tags: [community, protocol upgrade, core, crypto]
+author: sethforprivacy
+image: /blog/assets/2022-04-19-network-upgrade-july-2022/upgrades.png
+---v
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+The next network upgrade will take place on **16th July, 2022** (block 2,668,888). Network upgrades are technically hard-forks and require all participants in the Monero network to update their software. The new software will be released one month before the network upgrade, on the **~16th of June**.
```
Comment (SamsungGalaxyPlayer) [inline _posts/2022-04-19-network-upgrade-july-2022.md:14]: See comment on Flourine Fermi

File: _posts/2022-04-19-network-upgrade-july-2022.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+This network upgrade will introduce important new features like:
+
+- A ring-size increase from 11 to 16, increasing the base anonymity set (read: plausible deniability, or base privacy) of every transaction on the network
```
Comment (SamsungGalaxyPlayer) [inline _posts/2022-04-19-network-upgrade-july-2022.md:18]: A ring-size increase from 11 to 16, **the largest ever absolute increase in the base anonymity set** (read:

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]

File: _posts/2022-04-19-network-upgrade-july-2022.md (hunk 3 above)
Comment (sethforprivacy) [reply _posts/2022-04-19-network-upgrade-july-2022.md:18]: Love this language, good addition.

File: _posts/2022-04-19-network-upgrade-july-2022.md (hunk 1 above)
Comment (sethforprivacy) [reply _posts/2022-04-19-network-upgrade-july-2022.md:20]: Good catch, will match and be more eye catching on the important info.

**Comment by sethforprivacy** [comment]
Thanks for the recommended changes, pushed all of them @SamsungGalaxyPlayer.

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Review by erciccione — APPROVED** [review]

**Comment by binaryFate** [comment]
There's a typo "Flourine Fermi" -> "Fluorine Fermi"
@sethforprivacy can you fix?

**Pull request merged**